packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,480 @@
+});
+```
+
+```
+curl https://api.medplum.com/fhir/R4/Communication?id=example-ancestor-id&_revinclude=Communication:part-of
+```
+
+In this example we are searching for a communication with a given `id`. By including the `_revinclude` parameter, we also search for any communications whose `partOf` property references the communication with the given id. For more details on searching using linked resources, see the [Search](/docs/search/includes) documentation.
+
+If you are using `inResponseTo` to model a parent-child relationship, you can search for the parent communication and then iterate along the thread to get all of the communications.
+
+**Example: **
+
+```ts
+await medplum.searchResources('Communication', {
+  id: 'example-parent-id',
+  _revinclude:iterate: 'Communication:inResponseTo'
+});
+```
+
+In this example we are searching for a communication by `id`. After, we include the `_revinclude:iterate: 'Communication:inResponseTo` parameter, to retrieve all communications that reference the communication we are searching for. By including the `:iterate` modifier to recursively apply the reverse inclusion until no more resources are found. This first finds any direct responses to the initial communication. Then it iterates to find any communications that reference the direct response, and so on until it finds no more communications. For more details on using the `:iterate` modifier, see the [Search](/docs/search/includes) documentation.
+
+## Sorting Communications by Time
+
+Sorting communications by time or organizing messages chronologically is a common requirement when retrieving threaded communications. The communication resource includes two timestamp properties, `sent` and `received`, which represent the time that a message was sent or received. You can search for sorted results by including the special search parameter `_sort`, which allows you to specify a list of search parameters to sort by.
+
+Taking our example from earlier, we could search for a thread that uses `partOf` and sort the results with the below search.
+
+**Example: **
+
+```ts
+await medplum.searchResources('Communication', {
+  id: 'example-id',
+  _revinclude: 'Communication:part-of',
+  _sort: 'sent',
+});
+```
+
+This will return the communications sorted in ascending order. You can also sort by descending order and provide more than one argument to sort by in a comma separated list. The below example sorts the thread by descending time `sent` and then by `sender`.
+
+**Example: **
+
+```ts
+await medplum.searchResources('Communication', {
+  id: 'example-id',
+  _revinclude: 'Communication:part-of',
+  _sort: '-sent,sender',
+});
+```
+
+## Codesystems
+
+It is recommended to use LOINC or SNOMED codes to classify `Communication.category` and `Communication.topic`, but it is also possible to use custom coding internal to your practice.
+
+Below is an example of organizing a communication using all three of LOINC, SNOMED, and a custom code.
```

Comment:
We need a more opinionated rec here. Rec Snomed for practitioner roles and clinical specialty (link here: https://www.medplum.com/docs/careplans/tasks)


INternal code (most common), LOINC or SNOMED for topic